<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tyby File Comp v0.2b</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../usr/share/icons/hicolor/16x16/emblems/emblem-note.png</normaloff>../../../../../../usr/share/icons/hicolor/16x16/emblems/emblem-note.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Sortida">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>17</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Compara</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Fitxer_2">
       <property name="text">
        <string>File 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit2"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="text">
        <string>Cerca</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Fitxer_1">
       <property name="toolTip">
        <string>Or SHA512 hash</string>
       </property>
       <property name="text">
        <string>File 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="text">
        <string>Cerca</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionShow_info"/>
    <addaction name="actionThe_expert_hash"/>
    <addaction name="actionSortir"/>
   </widget>
   <widget class="QMenu" name="menuDaltonics">
    <property name="title">
     <string>Co&amp;lorblind</string>
    </property>
    <addaction name="actionActivate"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuDaltonics"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSortir">
   <property name="text">
    <string>&amp;Sortir</string>
   </property>
  </action>
  <action name="actionObrir">
   <property name="text">
    <string>Obrir</string>
   </property>
  </action>
  <action name="actionFitxer">
   <property name="text">
    <string>Fitxer</string>
   </property>
  </action>
  <action name="actionCadena">
   <property name="text">
    <string>Cadena</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>File info</string>
   </property>
  </action>
  <action name="actionActivate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Activate</string>
   </property>
  </action>
  <action name="actionShow_info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;info</string>
   </property>
   <property name="toolTip">
    <string>Will pop up file info if files matches</string>
   </property>
  </action>
  <action name="actionMd4">
   <property name="text">
    <string>M&amp;D4</string>
   </property>
   <property name="toolTip">
    <string>MD4 (Deprecated)</string>
   </property>
  </action>
  <action name="actionThe_expert_hash">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>The expert hash</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit1</tabstop>
  <tabstop>lineEdit2</tabstop>
  <tabstop>pushButton1</tabstop>
  <tabstop>pushButton2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionSortir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
